src/Repository/IdeaRepository.php

```diff
@@ -226,6 +226,7 @@ public function updateAuthorCategoryForIdeasOf(Adherent $adherent): void
             $qb->update()
                 ->set('idea.authorCategory', $qb->expr()->literal($categoryType))
                 ->where('idea.author = :adherent')
+                ->andWhere('idea.enabled = 0 OR idea.enabled = 1')
```

Comment:
In this case it may be better to use 'IN' instead of 'OR' ?
`
$qb->update()
                ->set('idea.authorCategory', $qb->expr()->literal($categoryType))
                ->where('idea.author = :adherent')
                ->andWhere($qb->expr()->in(''idea.enabled', [0, 1]))
                ->setParameter('adherent', $adherent)
                ->getQuery()
                ->execute()
            ;`

Reply:
We have no need of this line, because we manage Doctrine 'enabled' filter here https://github.com/EnMarche/en-marche.fr/pull/3392/files#diff-bc12631958b44351bc32d009a69a0ca0R42